cmake/public/LoadHIP.cmake

```diff
@@ -83,6 +83,13 @@ ELSE()
   SET(ROCRAND_PATH $ENV{ROCRAND_PATH})
 ENDIF()
 
+# MIOPENGEMM
+IF(NOT DEFINED ENV{MIOPENGEMM_PATH})
```

Comment:
Is this for preparation of future usages? (I don't see this lib getting used in this PR).

Reply:
The library is part of MIOpen, which we use. MIOpen's cmake config has trouble finding it unless we explicitly depend on it here.